src/redux/ActionCreators.js

```diff
@@ -1,18 +1,16 @@
 import * as ActionTypes from './ActionTypes';
 import {baseUrl} from '../shared/baseUrl';
-
 export const BooksLoading = () => ({
     type: ActionTypes.BOOKS_LOADING
 });
-
-
 export const fetchBooks = () => (dispatch) => {
-    dispatch(BooksLoading(true));
+    dispatch(BooksLoading());
     return fetch(baseUrl ,
```

Comment:
It is good practice to hold all api calls separately in its own js file and make it reusable.

Reply:
not using this file in the application.